src/config/mintr_version

```diff
@@ -1 +1 @@
-master
\ No newline at end of file
+mrc-1535
```

Comment:
This can be reverted to master when https://github.com/mrc-ide/mintr/pull/5 is merged

Reply:
Want to point this back and merge?